modules/Ui.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable  no-restricted-globals */
+/* eslint-disable  no-unused-vars */
+/* eslint-disable  no-undef */
+import ManageBooks from './Book_Management.js';
+
+const books = new ManageBooks();
+export default class Ui {
+  InitUi = () => {
+    books.display();
+    const navListBtn = document.querySelector('#nav-list');
+    const navAddNewBtn = document.querySelector('#nav-add-new');
+    const RemoveBtns = document.querySelectorAll('button');
+    const navContactBtn = document.querySelector('#nav-contact');
+    const AddBtn = document.querySelector('#add_Button');
+    const listSection = document.querySelector('#list');
+    const addNewSection = document.querySelector('#add-new');
+    const contactSection = document.querySelector('#contact');
+    const lastMod = document.querySelector('#last-modified');
+    setInterval(() => {
+      const date = luxon.DateTime.now();
+      lastMod.innerHTML = date.toLocaleString(luxon.DateTime.DATETIME_MED_WITH_SECONDS);
```

Comment:
- [ ] Luxon should  be imported as a module as well which will look something like below:
```
import { DateTime } from "luxon";
```

Reply:
@mikethreels i downloaded luxon as module ! but now I have to deal with the linters error of the js file that I downloaded ! so I will add disable linters for that  [file](https://moment.github.io/luxon/es6/luxon.js)

Reply:
Yes, for that file it is ok to disable the linters